packages/cli/commands/import/credentials.ts

```diff
@@ -29,87 +45,161 @@ export class ImportCredentialsCommand extends Command {
 		separate: flags.boolean({
 			description: 'Imports *.json files from directory provided by --input',
 		}),
+		id: flags.string({
```

Comment:
same as for other CLI command, would prefer the flag as `owner`, `user` or `ownerId/userId`